frontend/src/app/modules/result-selection/components/measurands/measurands.component.ts

```diff
@@ -16,11 +18,12 @@ export class MeasurandsComponent implements OnInit {
     isLoading: false,
     data: []
   });
+  aspectTypes$ = new ReplaySubject<PerformanceAspectType[]>(1);
```

Comment:
I think this would be better suited to be a BehaviorSubject because you can simply start with an empty list and avoid having an undefined variable. This could be beneficial as you only subscribe to the subject in the HTML of the child component.